pynamodb/exceptions.py

```diff
@@ -2,24 +2,26 @@
 PynamoDB exceptions
 """
 
+from typing import Any, Optional
+
 import botocore.exceptions
 
 
 class PynamoDBException(Exception):
     """
     A common exception class
     """
-    def __init__(self, msg=None, cause=None):
-        self.msg = msg or self.msg
+    def __init__(self, msg: Optional[str] = None, cause: Optional[Exception] = None):
+        self.msg = msg
         self.cause = cause
         super(PynamoDBException, self).__init__(self.msg)
```

Comment:
Let's modernize super syntax when we burn the bridges

Reply:
Yep, I haven't run the tools over this yet